README.md

```diff
@@ -29,30 +29,41 @@ or with an [Airflow DAG](https://github.com/lyft/amundsendatabuilder/blob/master
 Please note that the mock images only served as demonstration purpose.
 
 - **Landing Page**: The landing page for Amundsen including 1. search bars; 2. popular used tables;
-    
+
     ![](docs/img/landing_page.png)
-    
+
 - **Table Detail Page**: Visualization of a Hive / Redshift table
-    
+
     ![](docs/img/table_detail_page.png)
-    
+
 - **Column detail**: Visualization of columns of a Hive / Redshift table which includes an optional stats display
-    
+
     ![](docs/img/column_details.png)
-    
+
 - **Data Preview Page**: Visualization of table data preview which could integrate with [Apache Superset](https://github.com/apache/incubator-superset)
-    
+
     ![](docs/img/data_preview.png)
 
 ## Get Involved in the Community
 
-Want help or want to help? 
+Want help or want to help?
 Use the button in our [header](https://github.com/lyft/amundsenfrontendlibrary#amundsen) to join our slack channel. Please join our [mailing list](https://groups.google.com/forum/#!forum/amundsen-dev) as well.
 
+## Powered By
+
```

Comment:
I'd just add a description line. These are some of the organizations that are using Amundsen today. If your organization uses Amundsen, please file a PR and update this list.

Reply:
Other than that looks great!

Reply:
Doesn't Amundsen power the data discovery at X company? Seems like the relationship is going the wrong way with `Powered By`

Reply:
Kafka used `Powered By`(https://kafka.apache.org/powered-by) while Airflow used `Who uses Apache Airflow?`(https://github.com/apache/airflow#who-uses-apache-airflow).

Which one do you guys prefer?

Reply:
Updated:)